modules/ztp-image-based-upgrade-concept.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Using the `ImageBasedGroupUpgrade` CR for the image-based upgrade with {ztp}
+
+You can use a single resource, the `ImageBasedGroupUpgrade` CR, to define your stage transition policies and provide instructions for {cgu-operator} to reconcile the policies.
```

Comment:
We no longer use policies. I would say something like "to manage imaged-based upgrade on a group of cluster through all stages".